esmvalcore/_config/extra_facets/icon-mapping.yml

```diff
@@ -0,0 +1,23 @@
+# Extra facets for native ICON model output
+---
+
+ICON:
+  Amon:
+    clwvi: {raw_name: cllvi}
+  CF3hr:
+    clwvi: {raw_name: cllvi}
+  CFday:
+    clwvi: {raw_name: cllvi}
+  CFsubhr:
+    clwvi: {raw_name: cllvi}
+  E3hr:
+    clwvi: {raw_name: cllvi}
+  fx:
+    areacella:
+      latitude: grid_latitude
+      longitude: grid_longitude
+      raw_name: cell_area
+  SIday:
+    siconca: {raw_name: sic}
+  SImon:
+    siconca: {raw_name: sic}
```

Comment:
This is definitely not complete, but covers many variables (a list of the variables that we analyzed is given in the recipes here https://github.com/ESMValGroup/ESMValTool/tree/icon_eval/esmvaltool/recipes/icon). Most of the variables actually have the correct `short_name`.

Reply:
Nice. Perhaps add a short comment to that effect at the top of the file.

Reply:
I think `component` and `var_type` should appear in this file as well. This would simplify the recipes and it should be clear for every variable, right? You might want to experiment even with something like
```
  Amon:
    *:
      component: atm
      var_type: atm_2d_ml
```
though I suspect that ultimately this approach still means that more variables need to appear here.

Reply:
Perhaps that should even cover `version`.

Reply:
No, these the values for `component` and `var_type` are completely arbitrary, so it doesn't make sense to fix them here. Also, version could change as well, so we definitely want this in the recipe.

Reply:
Version could definitely change, that was really only a question.

But `component` and `var_type` are really completely arbitrary? `(Amon, tas)` will not always come from the atmosphere? That sounds surprising.

Reply:
The values are set by the person that runs the model, so we need them to stay here.

Reply:
Ok. We need a list with a short explanation of all supported extra facets. That should definitely be in the documentation part (will add a comment there), but might be repeated in the comment at the top of this file, at least stating which facets are expected here with a link to the full documentation.